build/Targets/Signing.Imports.targets

```diff
@@ -54,6 +54,8 @@
         <!-- Non-shipping binaries are simply signed with the Roslyn internal key. -->
         <Otherwise>
           <PropertyGroup>
+            <!-- Public sign on non-Windows as full signing isn't currently supported (https://github.com/dotnet/roslyn/issues/8210) -->
```

Comment:
Since the `<SignAssembly Condition="'$(OS)'=='Windows_NT'">true</SignAssembly>` conditional set was removed.